.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: ruby
 
+dist: trusty
```

Comment:
More of a learning point for me than a review point. But this itself has only got a year or so of shelf-life. I had a quick read into the docs provided and couldn't find how to switch to Xenial (yet).